pkg/cli/root.go

```diff
@@ -72,6 +74,10 @@ func newCommandRoot() *cobra.Command {
 			if err != nil {
 				return err
 			}
+			policyAPIClientset, err = policyapi.NewForConfig(k8sconf)
```

Comment:
will it work on a cluster that does not have ANPs supported? or will it crash with error?

Reply:
the analyzer should run on such cluster as well

Reply:
do we have an example of such cluster? I would test..